web_dashboard_tile/README.rst

```diff
@@ -36,6 +41,8 @@ Roadmap
 * Add icons.
 * Support client side action (like inbox).
 * Restore original Domain + Filter when an action is set.
+* Posibility to hide the tile based on a field expression.
```

Comment:
These are just comments under the Roadmap section. I just added them because I thought they might be a good addition to this module, but I haven't had the time to implement them right now. Also I didn't want to make a larger PR.